app/styles/includes/_includes.scss

```
@@ -0,0 +1,148 @@
+.product-big {
+    .product__list--big {
+        @include d-gird(1fr 1fr 1fr 1fr, 350px 216px 350px, 30px);
+
+        @include responsive(mobile) {
+            grid-template-columns: 1fr 1fr;
+            grid-template-rows: 30% 17% 17% 30%;
+            @include d-gird(1fr 1fr, 30% 17% 17% 30%, 30px);
```

```suggestion
            @include d-gird($columns: 1fr 1fr, $rows: 30% 17% 17% 30%, $gap: 30px);
```
Ngoài ra có thể khai báo như này để có thể dễ hình dung từng giá trị nó sẽ hỗ trợ cho thuộc tính nào trong trường hợp mixin có nhiều tham số